client/dashboard/agency-overview/index.tsx

```diff
@@ -5,18 +5,18 @@ import PageLayout from '../components/page-layout';
 export default function AgencyOverview() {
 	return (
 		<PageLayout
-			title={ __( 'Agency Overview' ) }
-			actions={
-				<>
-					<Button variant="primary" __next40pxDefaultSize>
+			headerProps={ {
```

Comment:
I passing `headerProps` in `PageLayout` as it made more sense to me than passing all `Header` props individually. I think it's clearer and easier to reason with.

Reply:
Why not use composition and just embed `Header` within `PageLayout`? I kind of hate all `somethingProps` props

Reply:
Alternatively `header={ <Header /> }` seems slightly better.

Reply:
I thought about this, but in that case we'd have to import the `Header` component in every page. Also it would result in different layouts if someone passed a different 'header`. Normally this shouldn't happen, but it's something that will not be done automatically.

I can use composition if you don't share my (small) concerns.

Reply:
- importing the component directly rather than `headerProps` is a good thing to me.
- someone could use a different header: a small downside indeed but I don't mind it personally.

Reply:
Okay. I'll update. Also, I just checked that I can use composition - there is no extra VStack.